www/js/app.js

```diff
@@ -1456,6 +1456,22 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                     // Configure home key press to focus #prefix only if the feature is in active state
                     if (params.useHomeKeyToFocusSearchBar)
                         iframeArticleContent.contentWindow.addEventListener('keydown', focusPrefixOnHomeKey);
+                    // Add event listener to iframe window to check for links to external resources
+                    iframeArticleContent.contentWindow.addEventListener('click', function (e) {
+                        if (e.target.tagName === 'A') {
+                            var href = e.target.getAttribute('href');
+                            // We assume that, if an absolute http(s) link is hardcoded inside an HTML string,
+                            // it means it's a link to an external website
+                            if (/^http/i.test(href)) {
```

Comment:
@mossroy Might it be worth including at least `^(?:http|ftp|mailto)` in this test? I can't think of others we are likely to encounter... except, maybe, `gopher`  🤣

Reply:
Well, I think we only need to handle the protocols that would otherwise be opened in the browser iframe.
`ftp` is not supported by recent browsers any more. But we could add it for older browsers anyway
Regarding `mailto`, `tel` or `geo` for example, browsers that know them will in already open another app. So we don't need to trap them.
I'll add the ftp protocol.
I think the other ones can be safely ignored :-)